tests/bitv/testfile-sign-extend-003.smt2

```diff
@@ -0,0 +1,4 @@
+(set-logic ALL)
+(declare-const x (_ BitVec 4))
+(assert (distinct ((_ extract 3 2) ((_ sign_extend 4) x)) ((_ extract 3 2) x)))
+(check-sat)
```

Comment:
This test is identical with `testfile-sign-extend-002.smt2`.

Reply:
Oops, this one was intended to test extraction of the high bits. Fixed.